### ISSUE TITLE
Alt text

### ISSUE THREAD

**Issue by farishkash** [dqbui/f1-3-c2p1-colmar-academy#4]
Each image should have descriptive alt text.  This helps the visually impaired through the use of screen readers know what each image is about.
